python/ccf/clients.py

```diff
@@ -417,13 +417,17 @@ def request(
         if self.signing_auth is not None:
             # Add content length of 0 when signing a GET request
             if request.http_verb == "GET":
-                if extra_headers.get("Content-Length") != "0":
+                LOG.error(extra_headers.get("Content-Length"))
```

Comment:
Don't think we want to be logging this everytime?

Reply:
Good spot! Thanks 👍